schema.graphql

```diff
@@ -233,3 +233,12 @@ type Solution @entity {
   txHash: Bytes!
   txLogIndex: BigInt!
 }
+
+type FleetDeployed @entity {
+  id: ID! # txHash + id
+  owner: Bytes!
+  fleet: [Bytes!]
```

Comment:
probably you want to map here an array of other entities, right?

Reply:
The fleet is really just a gnosis-safe address. At the time of `FleetDeployment`, there is no additional data available. Only later it would be good to have for each fleet also the orders and balances, but I guess this is done the best within the actual query, as orders and balances are well defined in the graph,